### ISSUE TITLE
Error loading image file on Linux systems

### ISSUE THREAD

**Issue by redbullx310** [JABeal/Fun-With-Pygame#2]
Getting an error when attempting to run Dino_Dodge.py

Traceback (most recent call last):
  File "/home/shane/Documents/Fun-With-Pygame/Dino_Dodge.py", line 355, in <module>
    startScreen()
  File "/home/shane/Documents/Fun-With-Pygame/Dino_Dodge.py", line 19, in startScreen
    character = makeSprite("Images/dino.gif")
  File "/home/shane/Documents/Fun-With-Pygame/pygame_functions.py", line 386, in makeSprite
    thisSprite = newSprite(filename, frames)
  File "/home/shane/Documents/Fun-With-Pygame/pygame_functions.py", line 107, in __init__
    img = loadImage(filename)
  File "/home/shane/Documents/Fun-With-Pygame/pygame_functions.py", line 285, in loadImage
    raise Exception("Error loading image: " + fileName + " - Check filename and path?")
**Exception: Error loading image: Images/dino.gif - Check filename and path?**

So basically whats happening here is the program is looking for a file called _dino.gif_, whereas in the Images folder, the file is actually named _Dino.gif_. The Windows environment is not a case-sensitive environment, which is why this issue will not appear when running the program in a Windows environment. 

However, the Linux environment is case-sensitive, which is why this issue is showing itself on a Linux machine. Linux sees dino.gif differently than Dino.gif. 

As a solution, all you need to do is change either the code or the filename to match capitalization; e.g. Dino.gif in code and in the Images folder.

**Comment by JABeal** [comment]
Thanks for the input! You actually caught an unnecessary file since Dino.gif is not actually used in the program. It was actually how I separated the gif into individual png images and interchanged them in each frame. I deleted the file Dino.gif and replaced it in the code with Images/dino1.png.

That should fix the problem. Let me know if you find any more issues whether Linux specific or not!
